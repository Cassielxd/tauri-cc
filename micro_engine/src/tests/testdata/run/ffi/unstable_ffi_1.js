Deno[Deno.internal].core.ops.op_ffi_load({ path: "", symbols: {} });
