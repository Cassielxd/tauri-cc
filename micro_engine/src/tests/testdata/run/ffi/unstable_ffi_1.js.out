Unstable API 'Deno.dlopen'. The --unstable flag must be provided.
